composer.json

```diff
@@ -14,7 +14,8 @@
       "ext-intl": "*",
       "delfimov/translate": "^2.4",
       "cultuurnet/search-v3": "^1.2",
-      "psr/container": "^1.0"
+      "psr/container": "^1.0",
+        "vimeo/psalm": "^4.4"
```

Comment:
In other repositories we use PHPStan for static analysis. I don't really have a preference for one or the other, but why switch tools now exactly? I'd prefer a consistent approach in dev tools like this so there's less friction working in multiple of these repositories (also coding standards for example but that's to be discussed when we pick up https://jira.uitdatabank.be/browse/III-3529).

If you believe Psalm's the better choice and can argument that, we can definitely use it but then I'd like to switch the rest to Psalm as well which will involve some (re)work in the repo's where we added PHPStan already.

Of course there's still https://jira.uitdatabank.be/browse/III-3558 to be picked up later to add PHPStan to other repo's and increase the levels, so it's not too late to switch of course.

Reply:
I think it's okay to go with PHPStan then cause people seem more familiar with it and I'll have to redo this PR anyway :D